<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassFilterSettingsPage</class>
 <widget class="QWidget" name="ClassFilterSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter classes</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="buttonBar" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btH">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btD">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lstClasses"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
